.github/workflows/cd-cloud-dev.yml

```diff
@@ -19,4 +19,4 @@ jobs:
           service_account_key: '${{ secrets.DEV_GCP_SA_KEY }}'
 
       - name: Deploy to dev GAE project
-        run: 'gcloud app deploy app.yaml --quiet --no-promote --version ${{ env.TAG }}'
```

Comment:
no, I think you want to keep `--quiet`, otherwise it defaults to interactive / gonna hang waiting for `[Y/n]`

Reply:
ohh is that what it does, ok i'll re add it